zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -215,13 +218,21 @@ protected ZkClient(IZkConnection zkConnection, int connectionTimeout, long opera
       throw new NullPointerException("Zookeeper connection is null!");
     }
 
+    _uid = UID.getAndIncrement();
+    if (LOG.isInfoEnabled()) {
+      LOG.info("ZkClient created with _uid {}, stacktrace {}", _uid, Thread.currentThread().getStackTrace());
```

Comment:
There are many time when I debugged, I wish to know where the zkclient is created when I look at the zkclient event thread logs. I think this is actually very useful. In fact, I would event want to add the _uid to each line of logs from Zkclient. It would save a lot time.

Reply:
Basically, for a log from a thread, such as ZkClient event thread or asycn retry thread, you have the thread id in the log. grepping thread id, you will find the _uid, grepping the _uid, you see the context of where the zkclient is created, such from zkhelixmanager or whatever other Helix java api. This is basically also the technique used to track the watcher leakage. 

The log would not be excessive as only one zkclient creation has 3 more lines.

For now, let me change it to debug, if this is ok. We can discussed adding this thing to callbackhandler and zkclient later.

Reply:
removed.